login/index.html

```diff
@@ -29,29 +31,38 @@
       </a>
     </header>
     <main class="login-main">
-      <form action="" method="post" class="login-form">
+      <form action="/items/" method="get" class="login-form">
```

Comment:
💊 제안
폼 태그의 action 속성은 form 데이터값을 보낼 URI로 아직 배우지 않은 API를 입력해주시는 속성입니다~
지금은 비워두시면됩니다~

[mdn form action](https://developer.mozilla.org/ko/docs/Web/HTML/Element/form#action)

```suggestion
      <form class="login-form">
```